pkg/api/validation/validation.go

```diff
@@ -52,6 +52,7 @@ const fieldImmutableErrorMsg string = `field is immutable`
 const isNotIntegerErrorMsg string = `must be an integer`
 
 var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
+var volumeModeErrorMsg string = validation.InclusiveRangeError(0, 0777)
```

Comment:
On Tue, Aug 02, 2016 at 08:52:11PM -0700, Tim Hockin wrote:
> > @@ -52,6 +52,7 @@ const fieldImmutableErrorMsg string = `field is immutable`
> >  const isNotIntegerErrorMsg string = `must be an integer`
> >  
> >  var pdPartitionErrorMsg string = validation.InclusiveRangeError(1, 255)
> > +var volumeModeErrorMsg string = validation.InclusiveRangeError(0, 0777)
> 
> I would not use this error.  This will print integer value, which is not

Oh, sorry, as I can't test this I assume this error would say that the value was
not within the range. I will change this, sorry

> friendly.  I would say something like "must be less than or equal to 0777" or
> "may only include bit in 0777" or simply "must be a valid permissions mode"

Yes, totally. I thought something like that was shown. But I will change this,
then :)